### DISCUSSION TITLE
How to make WindowDragArea dismiss "maximize" event

### DISCUSSION THREAD

**Discussion by birdcagedout** [flet-dev/flet#968] [Q&A]
i'm making a flet app with fixed size of width and height of window.
so i disabled page.resizable and page.window_maximizable to False

`page.window_width = 360`
`page.window_height = 400`
`page.window_resizable = False`
`page.window_maximizable = False`

and i want to hide the title bar and make my custom title baron the top
so i hid the title bar and buttons as follows:

`page.window_title_bar_hidden = True`
`page.window_title_bar_buttons_hidden = True`

but when i created and added my WindowDragArea control to page on the top as a custom title bar,
it doesnt stop the window maximize when double-clicking on it.

what i tried is to define a custom window event handler to steal the maximize event to dismiss it.
but it doesnt work.

`def win_event(e: ft.ControlEvent):`
`	if e.data == "maximize":`
`		return`
`page.on_window_event = on_window_event`

is there any way to cutomize WindowDragArea control? or at least stop it from maximize the window.
would anyone know how, let me know please. i appreciate it.

**Comment by Jamilton-Bento** [comment]
Pesquise na documentação, e eles tem um exemplo.


import flet as ft

def main(page: ft.Page):
    page.window_title_bar_hidden = True
    page.window_title_bar_buttons_hidden = True

    page.add(
        ft.Row(
            [
                ft.WindowDragArea(ft.Container(ft.Text("Drag this area to move, maximize and restore application window."), bgcolor=ft.colors.AMBER_300, padding=10), expand=True),
                ft.IconButton(ft.icons.CLOSE, on_click=lambda _: page.window_close())
            ]
        )
    )

ft.app(target=main)

**Comment by ndonkoHenri** [comment]
https://flet.dev/docs/controls/windowdragarea/#maximizable
